_posts/2019-08-19-this-week-in-rustfest-barcelona-scholarships.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "This Week in RustFest - Scholarships, Community Tickets and CFPs"
+description: "Announcing our Scholarship program, a CFP-extension & a one last round of Community Tickets"
+socialImageSrc: "/assets/posts/barcelona/barcelona-announcement.png"
+---
+
+We have a ton of exciting updates for y'all this week, so let's dive right in!
+
+## The RustFest Barcelona Scholarship program is now live!
+
+We had people tell us that they were sad RustFest didn't offer scholarships this year—we would have been just as sad, believe us, if that happened! But thankfully, this is not the case:
+
+<h3 style="text-align: center">
+  <a href="https://barcelona.rustfest.eu/scholarships/" style="opacity: 1">
+    Scholarship Applications for Barcelona now open!
+  </a>
+</h3>
+
+In our Scholarship program we are offering free tickets and travel support to members of underrepresented groups and people who would otherwise not be able to attend the conference. Based on the experience gathered during previous years' scholarship programs, we had many ideas for improvements of the process, which sadly delayed opening the application form a couple weeks, but there is still plenty of time to apply and we will start awarding the scholarships on a rolling basis in early September.
+
+Many of this year's improvements are centered around making the application process itself more inclusive, as part of this effort we are proud to announce our Scholarships are available in 3 languages: english, [spanish](https://barcelona.rustfest.eu/scholarships/es/) and [catalan](https://barcelona.rustfest.eu/scholarships/ca/)! Read all the other details on the [Scholarships page](https://barcelona.rustfest.eu/scholarships/) and send us feedback, we'd love to hear your feedback and ideas how to further improve our process!
+
```

Comment:
Add one more sentence to also apply here?

Reply:
Nah, there are plenty of links/CTAs in there already.